week1/delete_bst.py

```diff
@@ -0,0 +1,74 @@
+class Node:
+    def __init__(self, key):
+        self.key = key 
+        self.left = None
+        self.right = None
+ 
+ 
+def inorder(root):
+    if root is not None:
+        inorder(root.left)
+        print root.key,
+        inorder(root.right)
+def insert( node, key):
+    if node is None:
+        return Node(key)
+    if key < node.key:
+        node.left = insert(node.left, key)
+    else:
+        node.right = insert(node.right, key)
+    return node
+def minValueNode( node):
+    current = node
+    while(current.left is not None):
+        current = current.left 
+    return current 
```

Comment:
Give a blank line after each method so that code is readable

Reply:
done